packages/api-client/src/models/Assessment.ts

```diff
@@ -117,7 +117,7 @@ export function AssessmentFromJSONTyped(json: any, ignoreDiscriminator: boolean)
         'sourceType': json['sourceType'],
         'sourceId': json['sourceId'] == null ? undefined : json['sourceId'],
         'createdAt': (new Date(json['createdAt'])),
-        'snapshots': json['snapshots'] == null ? undefined : ((json['snapshots'] as Array<any>).map(SnapshotFromJSON)),
```

Comment:
@Jos-Jerus please don't remove these validations for arrays. If we receive some null values this cause a crash error in the UI